rwengine/include/data/VehicleGenerator.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef RWENGINE_VEHICLEGENERATOR_HPP
+#define RWENGINE_VEHICLEGENERATOR_HPP
+
+/**
+ * Stores information about where the game can generate vehicles.
+ */
+struct VehicleGenerator
+{
+	glm::vec3 position;
+	float heading;
+	/** ID of the vehicle to spawn, or -1 for random. */
+	int vehicleID;
+	int colourFG;
+	int colourBG;
+	bool alwaysSpawn;
+	short alarmThreshold;
+	short lockedThreshold;
```

Comment:
Please add a /// @todo to those variables we currently don't use (and possibly create an issue after merge!).
There will be hard to detect bugs later otherwise!